analytics_dashboard/static/sass/style-application.scss

```diff
@@ -7,7 +7,7 @@
 // config
 // --------------------
 @import "config-assets";
-@import "config-variables";
+@import "themes/credo/config-variables";
```

Comment:
It looks like maybe the `THEME_CSS` in `analytics_dashboard/settings/base.py` might accomplish this?